docs/markdown/Python-module.md

```diff
@@ -238,6 +238,63 @@ module.
 
 It exposes the same methods as its parent class.
 
+## A note about Debian installation paths
+
+Debian and derivatives (Ubuntu, Linux Mint etc.) [change default installation
+and import paths][debian-dist-packages] in system interpreter. The problem is,
+the way they do this is inconsistent with documented ways of discovering those
+paths inside Python. Specifically, the [`sysconfig` module][python-sysconfig]
+returns values that are not in [`sys.path`][python-sys-path]. This module is
+used internally by Meson and therefore the installation paths . But it is
```

Comment:
This sentence is confusing. "therefore in the installation paths"?  Or "therefore the installation paths are _something_"?

Reply:
Oops, this was misdrafted, the second part of the sentence was cut and pasted elsewhere. Sorry for this.